plugins/com.google.cloud.tools.eclipse.util.test/src/com/google/cloud/tools/eclipse/util/ArtifactRetrieverTest.java

```diff
@@ -30,10 +29,20 @@
   private ArtifactRetriever retriever = new ArtifactRetriever();
   
   @Test
-  public void testGetGuavaLatest() {
+  public void testGetServletLatest() {
+    // group and version IDs changed with 3.0 so this should be stable
+    ArtifactVersion servlet = retriever.getLatestArtifactVersion("javax.servlet", "servlet-api");
```

Comment:
About the unit tests making network connections, one option would be to think of these as integration tests and move them to the integration bundle? (I thought this can fall into the integration test category conceptually.)

Otherwise, we could create a new issue to mock this part later.